src/node.cc

```diff
@@ -1290,7 +1319,15 @@ static ExitCode StartInternal(int argc, char** argv) {
   }
 
   // Without --build-snapshot, we are in snapshot loading mode.
-  return LoadSnapshotDataAndRun(&snapshot_data, result.get());
+  if (!LoadSnapshotData(&snapshot_data)) {
+    return ExitCode::kStartupSnapshotFailure;
+  }
+  NodeMainInstance main_instance(snapshot_data,
+                                 uv_default_loop(),
+                                 per_process::v8_platform.Platform(),
+                                 result->args(),
+                                 result->exec_args());
+  return main_instance.Run();
```

Comment:
In the SEA config, when `useSnapshot` is set to `true` and `disableExperimentalSEAWarning` is `false`, the https://github.com/nodejs/node/blob/b5e16adb1d155759e7db405eead5a43cd425785d/lib/internal/main/embedding.js#L15 experimental SEA warning doesn't get emitted. Would it be possible to fix that?

Reply:
Yeah I think we can fix it by adding it to the mksnapshot.js